easybuild/easyconfigs/c/ccache/ccache-4.6.1.eb

```diff
@@ -0,0 +1,48 @@
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# Updated to 4.6.1: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeNinja'
+
+name = 'ccache'
+version = '4.6.1'
+
+homepage = 'https://ccache.dev/'
+description = """Ccache (or “ccache”) is a compiler cache. It speeds up recompilation by
+caching previous compilations and detecting when the same compilation is being done again"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/ccache/ccache/releases/download/v%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['59b28a57c3a45e48d6049001999c9f94cd4d3e9b0196994bed9a6a7437ffa3bc']
+
+osdependencies = [('glibc-static', 'libc6-dev', 'glibc-devel')]
+
+local_gccver = '11.2.0'
+builddependencies = [
+    ('GCC', local_gccver),
+    ('CMake', '3.22.1', '', ('GCCcore', local_gccver)),
+    ('Ninja', '1.10.2', '', ('GCCcore', local_gccver)),
+    ('zstd', '1.5.0', '', ('GCCcore', local_gccver)),
+    ('pkg-config', '0.29.2', '', ('GCCcore', local_gccver))
+]
+
+dependencies = [
+    ('hiredis', '1.0.2', '', ('GCCcore', local_gccver)),
```

Comment:
@sassy-crick This implies that `GCCcore` should be used as a toolchain (instead of system), since both `ccache` and `hiredis` will both "pull in" `GCCcore` when their module is loaded.